pkg/server/server.go

```diff
@@ -57,6 +56,7 @@ import (
 	kcpfeatures "github.com/kcp-dev/kcp/pkg/features"
 	kcpserveroptions "github.com/kcp-dev/kcp/pkg/server/options"
 	"github.com/kcp-dev/kcp/pkg/sharding"
+	"github.com/kcp-dev/kcp/pkg/virtual/framework/forwardingregistry"
```

Comment:
😱 – why?!?!

Reply:
am not conformatable with having this client here.

Actually, if you rebase onto main, the client creation has been moved into the virtual workspace apiservers. I.e. you could instantiate the custom client there, which would be much cleaner.

Reply:
Yes, totally agree. It was meant to be temporary to have your feedback and have it moved to a proper package.